docs/concepts/flox-vs-containers.md

```diff
@@ -0,0 +1,452 @@
+---
+Title: Flox vs. container workflows
+Description: How does a Flox workflow differ from a container workflow?
+---
+
+# Flox vs. container workflows
+
+Containers are everywhere these days.
+They're the de-facto deployment method in the industry,
+and they're often used for local development as well to ensure that every
+developer gets the same environment.
+
+Flox gives you these same benefits without many of the papercuts,
+but the workflow is slightly different so it's worth exploring.
+
+## Create a new development environment
+
+Let's say you've created a new directory for your project,
+`myproject`,
+and entered it:
+
+```bash
+$ mkdir myproject
+$ cd myproject
+```
+
+=== "Flox"
+
+    Create a Flox environment for the project via [`flox init`][init]:
+
+    ```bash
+    $ flox init
+    ```
+
+    This creates a `.flox` directory in `myproject`.
+    At this point you can enter the environment,
+    but it doesn't provide any new packages or functionality.
+
+=== "Containers"
+
+    Create a new `Dockerfile`.
+
+    ```bash
+    $ touch Dockerfile
+    ```
+
+    At this point the `Dockerfile` isn't really useable since it doesn't say which
+    image to build on top of (there's no `FROM` line).
+    Let's pick the latest Ubuntu LTS release.
+
+    ```bash
+    $ echo "FROM ubuntu:noble" >> Dockerfile
+    ```
+
+    This already adds some variability in that which image this refers to may
+    change from moment to moment.
+
+---
+
+## Add packages
+
+Now let's install some packages.
+In a typical project there's usually different subsets of packages:
+
+- Packages you don't care about too much, so the latest version will suffice.
+- Packages whose versions you want pinned to a recent version.
```

Comment:
**nitpick:** This is a stylistic quibble, but I'd use "of which" instead of "whose" here given that we're not talking about persons. That may come off sounding a bit stodgy though, and I doubt anyone will misunderstand "whose." Your call on whether or not to change it!
```suggestion
- Packages the versions of which you want pinned to a recent version.
```